srml/democracy/src/lib.rs

```diff
@@ -803,7 +802,7 @@ impl<T: Trait> Module<T> {
 	fn do_vote(who: T::AccountId, ref_index: ReferendumIndex, vote: Vote) -> Result {
 		ensure!(Self::is_active_referendum(ref_index), "vote given for invalid referendum.");
 		if !<VoteOf<T>>::exists(&(ref_index, who.clone())) {
-			<VotersFor<T>>::mutate(ref_index, |voters| voters.push(who.clone()));
+			safe_append_map(VotersFor(PhantomData::<(T,)>), ref_index, who.clone());
```

Comment:
@bkchr this was my question. I assumed something like below would work: 
```suggestion
			safe_append_map(VotersFor, ref_index, who.clone());
```